src/main/java/com/blackducksoftware/integration/fortify/batch/util/MappingParser.java

```diff
@@ -405,4 +406,16 @@ private CreateApplicationRequest createApplicationVersionRequest(String fortifyP
                 new CreateApplicationRequest.Project("", fortifyProjectName, "Built using API", TEMPLATE), TEMPLATE);
     }
 
+    /**
+     * Delete the Fortify application version
+     *
+     * @param applicationId
+     * @throws IOException
+     * @throws IntegrationException
+     */
+    private int deleteApplicationVersion(int applicationId) throws IOException, IntegrationException {
```

Comment:
int is kind of opaque here. Im not sure if it was added in this review, but in general I would probably go down one of two paths:

- if there are only success and failure conditions, i would use a boolean
- if there are multiple conditions i would use an enum instead of just an int